### ISSUE TITLE
A more straightforward node bar UI

### ISSUE THREAD

**Issue by leafcutterant** [m2049r/xmrwallet#626]
When you open the app, if all goes right, a node is loaded and the name displayed in the node bar. Currently, if you tap on the node name, it disconnects, searches for nodes and reconnects. It's not obvious that that will happen if you tap it. It often happened that I tapped on the node name because my instinct was that that's how I actually connect, or that's how I enter the node list. Due to how fickle node availability is, I got burnt by this many times (connection was established, I tapped on it, but after the disconnect I couldn't reconnect and I couldn't use the wallet for hours on).

Bottom line: an established node connection is too precious to let the user lose it due to vague UI choices.

I propose modifying the opening screen node bar UI as follows: 
* tapping the node name either does nothing, or opens the node list
* a (static) circular arrow icon is displayed next to the magnifier icon, and the disconnect-search-reconnect process can be triggered by tapping this
